src/content/docs/infrastructure/host-integrations/host-integrations-list/dojo-io-integration.mdx

```diff
@@ -0,0 +1,30 @@
+---
+title: Dojo.io integration
+tags:
+    - dojo.io integration
+    - dojo integration
+    - new relic integrations
+metaDescription: "The New Relic Dojo.io integration sends performance metrics and inventory data from your Dojo.io framework to the New Relic platform."
+---
+import dojodashboard from 'images/infrastructure_screenshot-crop_dojo-dashboard.png'
+
+With the help of New Relic's Dojo dashboard, you can track your application's core web vitals, user time on the site, and browser performance metrics like Initial page load, route change, throughput and page views with Javascript errors. By using our browser monitoring agent, the Dojo dashboard lets you view all your most important data in one place.
```

Comment:
```suggestion
With the help of New Relic's Dojo dashboard, you can track your application's core web vitals, how much time your users spend on your site, and critical browser performance metrics like initial page load and pages with Javascript errors. Built with our browser monitoring agent, the Dojo dashboard lets you view all your most important data in one place.
```

Reply:
I'm not sure if "how much time" clause is better than the initial draft, but I think it teases out what exactly that phrase means